website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 90
+---
+## General
+
+### How does a provisioner decide to manage a particular node?
+Karpenter only provisions nodes for pods with a status condition `Unschedulable=True`
+Karpenter will only take action on nodes that it provisions.
+All nodes launched by Karpenter will be labeled with `karpenter.sh/provisioner-name`.
+
+### What cloud providers are supported?
+AWS is the first cloud provider supported by Karpenter, although it is designed to be used with other cloud providers as well.
+See [[Cloud provider]({{< ref "/docs/concepts/#cloud-provider" >}}) for details.
+
+### What deployment methods are supported by Karpenter?
```

Comment:
I'd leave this out -- we have other easily discoverable mechanisms for this information.